utils/obsproc/superob.h

```diff
@@ -8,11 +8,47 @@
 
 namespace gdasapp {
   namespace superobutils {
+    // Function to compute distance between two points
+    struct GeoPoint {
+        double latitude;
+        double longitude;
+    };
+    const double EARTH_RADIUS_KM = 6371.0;
```

Comment:
use `atlas::Earth` ... or something ... But the earht geometry is already defined in atlas, so no need to redefine the earht radius again.

Reply:
Thank you for the suggestion, theses are very useful information. I will check functions available in altas and eckit.